modules/web_hosting/network.tf

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#########################################################################
+# vpc-xlb - VPC Network & Subnests
+#########################################################################
+
+resource "google_compute_network" "vpc-xlb" {
```

Comment:
Is there a reason why we are not using the Cloud Foundation Toolkit module here?

Reply:
It was just more complex to use subnets list objects when referencing its elements to  google_compute_instance resources. I can replace it and will refer just network instead of subnet.

Reply:
Unfortunately, Subnetwork should be specified for custom subnet mode network within google_compute_instance resource. Thus will be opting out of Cloud Foundation Toolkit module here.